VAMobile/src/screens/HomeScreen/ProfileScreen/ProfileScreen.tsx

```diff
@@ -6,45 +6,40 @@ import { StackScreenProps } from '@react-navigation/stack'
 import { useAuthorizedServices } from 'api/authorizedServices/getAuthorizedServices'
 import { useServiceHistory } from 'api/militaryService'
 import { usePersonalInformation } from 'api/personalInformation/getPersonalInformation'
-import { Box, ChildTemplate, ErrorComponent, LargeNavButton, LoadingComponent, NameTag, TextView } from 'components'
+import {
+  Box,
+  CategoryLandingAlert,
+  ChildTemplate,
+  ErrorComponent,
+  LargeNavButton,
+  LoadingComponent,
+  NameTag,
+  TextView,
+} from 'components'
 import { NAMESPACE } from 'constants/namespaces'
 import { HomeStackParamList } from 'screens/HomeScreen/HomeStackScreens'
-import { ScreenIDTypesConstants } from 'store/api/types'
-import { useError, useRouteNavigation, useTheme } from 'utils/hooks'
+import { DowntimeFeatureTypeConstants, ScreenIDTypesConstants } from 'store/api/types'
+import { useDowntime, useRouteNavigation, useTheme } from 'utils/hooks'
 
 type ProfileScreenProps = StackScreenProps<HomeStackParamList, 'Profile'>
 
 function ProfileScreen({ navigation }: ProfileScreenProps) {
+  const theme = useTheme()
+  const navigateTo = useRouteNavigation()
+  const { t } = useTranslation(NAMESPACE.COMMON)
+
   const {
     data: userAuthorizedServices,
     isLoading: loadingUserAuthorizedServices,
     error: getUserAuthorizedServicesError,
     refetch: refetchUserAuthorizedServices,
   } = useAuthorizedServices()
-
-  const {
-    isLoading: loadingServiceHistory,
-    error: serviceHistoryError,
-    refetch: refetchServiceHistory,
-  } = useServiceHistory()
-  const navigateTo = useRouteNavigation()
-  const theme = useTheme()
-  const { t } = useTranslation(NAMESPACE.COMMON)
   const { data: personalInfo } = usePersonalInformation()
+  const { isLoading: loadingServiceHistory, isError: serviceHistoryError } = useServiceHistory()
 
-  /**
-   * Function used on error to reload the data for this page. This combines all calls necessary to load the page rather
-   * than checking the needsDataLoad flag because if something went wrong we assume we want to reload all of the necessary data
-   */
-  const getInfoTryAgain = (): void => {
-    refetchUserAuthorizedServices()
-    if (userAuthorizedServices?.militaryServiceHistory && serviceHistoryError) {
-      refetchServiceHistory()
-    }
-  }
+  const serviceHistoryInDowntime = useDowntime(DowntimeFeatureTypeConstants.militaryServiceHistory)
```

Comment:
I think we should maybe make a code upkeep ticket to rethink our current Downtime structure. @theodur can you follow up with me about this on Monday?